workflow/scripts/postprocess_diffexp.py

```
@@ -1,5 +1,9 @@
+import sys
+sys.stderr = open(snakemake.log[0], "w")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)